### ISSUE TITLE
Disconnect BT Special Action Not Working with Sony Dualshock 4 Wireless USB Adaptor

### ISSUE THREAD

**Issue by ajhix36** [Ryochan7/DS4Windows#70]
Hello, thank you for your continued support on this software!

I've noticed that I am unable to get the PS4 Controller to disconnect from BT using a special action.
If I hit stop, it disconnects. If I let it idle, it disconnects. But I can not get the Special Action to trigger.

I would like to be able to disconnect in order to turn off the controller faster than holding guide down for 15 seconds.

Thank you!

**Comment by Ryochan7** [comment]
I remembered that not allowing the Disconnect BT special action to work was a  conscious decision but I could not remember why. Looking back at previous issues has shown why I made the initial decision to not allow Disconnect BT to work for the Sony Dongle. The problem is that the Sony Dongle will spam the last controller state until a controller is connected again. Having the PS button being seen as pressed can cause problems with some applications such as Steam.

Some recent refactors could work around some issues when using Exclusive Mode and XInput emulation. Nothing can be done if Exclusive Mode is not being used though. I am not sure if this should be looked at again or if the Disconnect BT special action should stay dedicated to only working with Bluetooth connections.

https://github.com/Ryochan7/DS4Windows/issues/13
